package.json

```diff
@@ -74,5 +74,12 @@
 		"test": "vitest"
 	},
 	"type": "module",
-	"version": "0.0.17"
+	"version": "0.0.17",
+	"files": [
+		"lib/",
+		"package.json",
+		"CHANGELOG.md",
```

Comment:
Does this need to be here? I imagine the README.md is enough info that folks can get to docs from there, yeah?

```suggestion
```

If we're optimizing for package size then I imagine changelogs -which can be pretty huge- might not be necessary?

Reply:
No, changelog isn't required at all. I know some packages like to include it anyway though.

I included it here just because it was included in the latest release.